plugin/vim-jdb.vim

```diff
@@ -186,36 +276,57 @@ function! s:detach()
 endfunction
 
 function! s:breakpointOnLine(fileName, lineNumber)
-  "TODO check if we are on a java file and fail if not
-  let fileName = s:getClassNameFromFile(a:fileName)
-  "TODO store command temporary if not already connected
-  call ch_sendraw(s:channel, "stop at " . fileName . ":" . a:lineNumber . "\n")
+  if s:channel != ''
+    let l:fileName = s:getClassNameFromFile(a:fileName)
+    call ch_sendraw(s:channel, "stop at " . l:fileName . ":" . a:lineNumber . "\n")
+  endif
 endfunction
 
 function! s:clearBreakpointOnLine(fileName, lineNumber)
-  "TODO check if we are on a java file and fail if not
-  let fileName = s:getClassNameFromFile(a:fileName)
-  "TODO store command temporary if not already connected
```

Comment:
For me, both todo's are not fixed. I would like to keep them until they are fixed or decided not to be implemented.